joomla/joomla-cms
# [4.0][a11y] Amenu

Pull Request for Issue # .

### Summary of Changes

adding keyboard navigation to menu

### Testing Instructions

create different types of menus (with sub-menus and without)
Reach (with 'tab') the menu under test and check the following behavior:

- right arrow - next item (same level). If last - back to first item
- left arrow - prevision item (same level) . if first - go to last item
- down arrow - if sub-menu exist - go to sub level, first item
                      - if not - behave as right arrow
- up arrow - if first item in sub-menu - go to parent item
                 - if not first item or no parent item - same as left arrow

### Expected result

The behavior was explained above 

### Actual result



### Documentation Changes Required
